systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -92,8 +92,8 @@ void testDeployAndUnDeployCruiseControl() throws IOException {
         LOGGER.info("Verifying that there is no configuration to CruiseControl metric reporter in Kafka ConfigMap");
         assertThrows(WaitException.class, () -> CruiseControlUtils.verifyCruiseControlMetricReporterConfigurationInKafkaConfigMapIsPresent(CruiseControlUtils.getKafkaCruiseControlMetricsReporterConfiguration(testStorage.getNamespaceName(), testStorage.getClusterName())));
 
-        // https://github.com/strimzi/strimzi-kafka-operator/issues/8864
-        if (!Environment.isKRaftModeEnabled() && !Environment.isUnidirectionalTopicOperatorEnabled()) {
+        // TODO https://github.com/strimzi/strimzi-kafka-operator/issues/8864
+        if (!Environment.isKRaftModeEnabled()) {
```

Comment:
I guess this will fail from now on, because it will not be skipped, or?

Reply:
Correct, this is also failing. I'll comment these if statements and leave the fix to you as part of #8864.

Reply:
Yeah, great
Thanks :)